client.go

```diff
@@ -67,6 +71,42 @@ type zulipUserNotification struct {
 	zulipAPIURL string
 }
 
+// implements streamMessage
+type zulipStreamMessage struct {
+	botUsername string
+	zulipAPIURL string
+}
+
+func (zsm *zulipStreamMessage) postToTopic(ctx context.Context, botPassword, message string, stream string, topic string) error {
```

Comment:
This is being used to post to 397 Bridge to welcome new recursers to Pairing Bot.